locustempus/main/views.py

```diff
@@ -572,72 +572,40 @@ def get(self, request, *args, **kwargs) -> HttpResponse:
         project: Any = get_object_or_404(
             Project.objects.filter(course=course.pk),
             pk=kwargs.get('project_pk', None))
+
+        new_project = False
```

Comment:
The new_project flow is a bit awkward...maybe we should chat a bit more about it?

Reply:
Sounds good, I'll book some time.